client/src/featureform/serving.py

```diff
@@ -17,7 +17,7 @@
 
 class Client:
 
-    def __init__(self, host=None, local=False, tls_verify=True, cert_path=None):
+    def __init__(self, host=None, local=False, insecure=False, cert_path=None):
```

Comment:
make sure that serving client and resource client are consistent with this

Reply:
Yeah i checked this and they are consistent. All the tls_Verify variables have been changed to insecure variables for consistency.